core/components/atoms/dropdown/__stories__/variants/WithErrorTempate.story.jsx

```diff
@@ -0,0 +1,73 @@
+import * as React from 'react';
+import { Dropdown, Heading } from '@/index';
+import { Uncontrolled, Controlled } from '../_common_/types';
+
+// CSF format story
+export const withErrorTemplate = () => {
+  const options = [];
+
+  const getSearchedOptions = (options, searchTerm) => {
+    const result = options.filter((option) => option.label.toLowerCase().includes(searchTerm.toLowerCase()));
+    return result;
+  };
+
+  const fetchOptions = (searchTerm) => {
+    const searchedOptions = searchTerm ? getSearchedOptions(options, searchTerm) : options;
+    return new Promise((resolve) => {
+      this.window.setTimeout(() => {
+        resolve({
+          searchTerm,
+          options: searchedOptions,
+          count: searchedOptions.length,
+        });
+      }, 1000);
+    });
+  };
+
+  const errorTemplate = <Heading appearance={'disabled'}>No data is fetched.</Heading>;
+
```

Comment:
Please use the proper error template as provided in design

Reply:
Done